en/peripherals/rtk_gps_hex_hereplus.md

```diff
@@ -38,3 +38,11 @@ The Here+ GPS pinout is provided below. This can be used to help modify the conn
 | 6   | BUTTON        | -   | -                 |
 | 7   | BUTTON_LED    | -   | -                 |
 | 8   | GND           | 6   | GND               |
+
+### Pin Removal instructions.  
+
+Pull the white piece of plastic between the metal stripes (I use a straight pin, to lift it up.
```

Comment:
Missing end bracket and new sentence not capitalised. How about replace these two lines with:

Pull the white piece of plastic between the metal stripes (e.g. use a straight pin, to lift it up) and then pull the pin out of the back side of the connector.